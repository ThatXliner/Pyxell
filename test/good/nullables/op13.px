
func f(Int x) [Int]? def
    print x
    x *= 2
    return x >= 0 ? [x] * x : null

print f(100)?.length, f(-1)?.length
print f(-3)?[0], f(3)?[0]
