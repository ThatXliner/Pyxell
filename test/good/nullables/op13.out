100
-1
200 null
-3
3
null 6
